### PR TITLE
 Cleanup patterns

### PR DESCRIPTION
**Pull request by Deewarz**
### Summary
- Remove unwanted `Addr` suffix on some names
- Use `hook::get_pattern()` instead of `hook::pattern().get_first()`
- Remove useless `utils/hooking/hooking.h` import (already imported in `pattern.h`)
- Moving patterns to dedicated file if it was not there
- Group by name and alphabetical order
- Remove absolute import in sdk files
- Remove sdk relative import in project (outside sdk folder)
- Always use `__` (2 underscores) between class name and attributes

### CODE REVIEW

File: code/client/src/sdk/mafia/framework/c_mafia_dbs.h [hunk 1]
```diff
@@ -2,26 +2,18 @@
...
-            }
+            void GetTablesDatabase(ue::sys::sodb::C_SysODB *);
 
             ue::C_WeakPtr<mafia::framework::C_VehiclesDatabase> GetVehiclesDatabase();
         };
 
-        static C_MafiaDBs *GetMafiaDBs() {
-            const auto pattern = hook::get_opcode_address("E8 ? ? ? ? 48 8D 55 87 48 8B C8");
-            return hook::call<C_MafiaDBs *>(pattern);
-        }
```
Comment (Segfaultd) [inline code/client/src/sdk/mafia/framework/c_mafia_dbs.h:25]: I have a doubt on this. It was a static method. You're making a non static method of it, and also forgot passing the this pointer. Call will always fail.

@Greavesy1899 can you confirm that this setup works?

**Review by Segfaultd — CHANGES_REQUESTED** [review]

File: code/client/src/sdk/mafia/framework/c_mafia_dbs.h (hunk 1 above)
Comment (Greavesy1899) [reply code/client/src/sdk/mafia/framework/c_mafia_dbs.h:25]: I'm certain this should remain static. It's a getter function to fetch a global variable, so I'd keep it static.

File: code/client/src/sdk/mafia/streaming/c_streaming_module.h
Comment (zpl-zak) [inline code/client/src/sdk/mafia/streaming/c_streaming_module.h:1]: keep as sdk, likewise other paths should also start using absolute paths at some point.

**Review by zpl-zak — CHANGES_REQUESTED** [review]

File: code/client/src/sdk/mafia/streaming/c_streaming_module.h
Comment (Deewarz) [reply code/client/src/sdk/mafia/streaming/c_streaming_module.h:1]: @Segfault told me the opposite :p
Can we agree on a way to do?

Comment (Deewarz) [reply code/client/src/sdk/mafia/streaming/c_streaming_module.h:1]: @Segfaultd told me the opposite :p
Can we agree on a way to do?

Comment (zpl-zak) [reply code/client/src/sdk/mafia/streaming/c_streaming_module.h:1]: relative paths are ugly, especially when they need to step out more than 2 times out of current dir, like `../../../foo/bar.h`, where a single `sdk/foo/bar.h` would usually suffice, it's a suggestion though.

Comment (Segfaultd) [reply code/client/src/sdk/mafia/streaming/c_streaming_module.h:1]: I'm with the idea of *at scale* taking the SDK out of MafiaMP project and making it standalone for any other project that might use it (trainer etc...).

So if we can minimize the setup required to use it (and by setup I mean creating absolute paths), isn't it better?

Open discussion @zpl-zak

Comment (zpl-zak) [reply code/client/src/sdk/mafia/streaming/c_streaming_module.h:1]: Ye in that case it's fine, but let's only do that in SDK code only, please.

Just to add I'm ok with it when the included file is next to you `#include "other.h"` or above dir `#include "../baz.h"`, but it gets messy in some other cases.

**Review by zpl-zak — APPROVED** [review]

**Review by Segfaultd — APPROVED** [review]

**Pull request merged**